pyiron/atomistics/structure/atoms.py

```diff
@@ -331,6 +330,51 @@ def set_high_symmetry_points(self, new_high_symmetry_points):
             raise ValueError("has to be dict!")
         self._high_symmetry_points = new_high_symmetry_points
 
+    def add_high_symmetry_points(self, new_points):
+        """
+        Adds new points to the dict of existing high symmetry points.
+
+        Args:
+            new_points (dict): Points to add
+        """
+        if self.get_high_symmetry_points() is None:
+            raise AssertionError("Construct high symmetry points first. Use self.band_structure_calculations().")
+        else:
+            self._high_symmetry_points.update(new_points)
+
+    def get_high_symmetry_path(self):
+        """
+        Path used for band structure calculations
+
+        Returns:
+            dict: dict of pathes with start and end points.
+
+        """
+        return self._high_symmetry_path
+
+    def _set_high_symmetry_path(self, new_path):
+        """
+        Sets new list for the high symmetry path used for band structure calculations.
+
+        Args:
+            new_path (dict): dictionary of lists of tuples with start and end point.
+                E.G. {"my_path": [('Gamma', 'X'), ('X', 'Y')]}
+        """
+        self._high_symmetry_path = new_path
+
+    def add_high_symmetry_path(self, path):
+        """
+        Adds a new path to the dictionary of pathes for band structure calculations.
+
+        Args:
+            path (dict): dictionary of lists of tuples with start and end point.
+                E.G. {"my_path": [('Gamma', 'X'), ('X', 'Y')]}
+        """
+        if self.get_high_symmetry_path() is None:
+            raise AssertionError("Construct high symmetry path first. Use self.band_structure_calculations().")
+        else:
+            self._high_symmetry_path.update(path)
```

Comment:
It would be helpful if the function also checks the content of the dictionary (especially because "dictionary of lists of tuples is quite complex), e.g.

```
for value in path.values():
    for index in range(len(value)-1):
        if value[index][1] != value[index+1][0]:
            raise ValueError('Path not connecting at {} and {}'.format(' '.join(value[index]), ' '.join(value[index+1]))
```

Reply:
Nice idea, but the path do not need to be connected. A path `G->H->N, G->P` is also valid. I will ad a check, whether the given points of the path are also valid keys of the high symmetry points.

Reply:
done